benchmarks/kotlinx_benchmark.toml

```diff
@@ -0,0 +1,25 @@
+[general]
+    # benchmark name
+    name = 'kotlinx'
```

Comment:
Why do we need this here? It's a library for benchmarks, not a test suite for benchmarking

Reply:
we will have a category `tools` also (frameworks/libs/etc)